pkg/schedule/schedulers/balance_range.go

```diff
@@ -366,6 +366,7 @@ type balanceRangeScheduler struct {
 	handler       http.Handler
 	filters       []filter.Filter
 	filterCounter *filter.Counter
+	plan          *balanceRangeSchedulerPlan
```

Comment:
How about changing the plan to solver since the plan is used for diagnostics, usually.